guides/common/modules/proc_creating-a-personal-access-token.adoc

```diff
@@ -0,0 +1,37 @@
+[id="creating-a-personal-access-token_{context}"]
+= Creating a {PAT}
+
+Use this procedure to create a {PAT}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Users*.
+. Select a user for which you want to create a {PAT}.
+. On the *{PAT}s* tab, click *Add {PAT}*.
+. Enter a *Name* for you {PAT}.
+. Optional: Select the *Expires* date to set an expiration date.
+If you do not set an expiration date, {PAT} will never expire unless revoked.
+. Click Submit.
+You now have the {PAT} available to you on the *{PAT}s* tab.
+. Click *Copy to clipboard* to copy your {PAT}.
++
+[IMPORTANT]
+====
+Ensure to store your {PAT} as you won't be able to access it again after you leave the page or create a new {PAT}.
```

Comment:
I'd like to avoid using shortened forms as they sound less official (I also don't like me sounding like a bureaucrat
 here). Maybe we can create an issue to have further discussion and then add the agreed-upon rules to README?

Reply:
That being said, I had a brain-freeze and didn't realize I was the one using shortened form. My apologies. No discussion needed.